docs/personal-data.md

```diff
@@ -57,12 +57,18 @@ These are all the datatypes that are used in the events (i.e. the `payload`). Da
 
 A list of all the [Identity events]((../src/Surfnet/Stepup/Identity/Event/) in stepup and the data that these events store in the `payload`.
 
-[AppointedAsRaaEvent](../src/Surfnet/Stepup/Identity/Event/AppointedAsRaaEvent.php)
+[~~AppointedAsRaaEvent~~](../src/Surfnet/Stepup/Identity/Event/AppointedAsRaaEvent.php) deprecated
```

Comment:
Suggestions:
- The strikethough is a nice touch to indicate the event should not be used any more
- I'd like to see a reference to what event superseded the deprecation

Maybe format it like this?
[~~AppointedAsRaaEvent~~](../src/Surfnet/Stepup/Identity/Event/AppointedAsRaaEvent.php) 
Status: deprecated
Superseded by: [AppointedAsRaaEvent](../src/Surfnet/Stepup/Identity/Event/AppointedAsRaaEvent.php)
```

Reply:
Yes, good idea this will be more clear and informational for the reader